packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -585,6 +234,43 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
       }
     );
   }
+
+  /**
+   *
+   */
+  private createLine() {
+    const lineId = uuid.v4();
+
+    const lineInstance = new Line(
+      // dummy values
+      this.webex.internal.device.userId,
+      this.webex.internal.device.url,
+      lineId,
+      '131212345',
+      '212343435',
+      LineStatus.INACTIVE,
+      ['add1', 'add2'],
+      '321232343',
+      '00',
```

Comment:
what are these numbers here?

Reply:
These are the temporary dummy values for testing the creation of line object. We won't need it once we get config file from DMS.

Reply:
@meta-dipanshu-sharma as of now we aren't going to contact DMS to get the device configuration. So we don't need to keep dummy data. When the Mobius starts supporting multiple lines (which will take some time), we can add this back

Reply:
A few fields here will get populated once we get a registration response from Mobius so we can remove those from here:

"lastSeen" : "2023-04-18T13:04:53Z",
"addresses" : [ "sip:ce4itauc6t_GM1ILJVUKPEC@[64941297.int10.bcld.webex.com

  "keepaliveInterval" : 30,
  "callKeepaliveInterval" : 900,
  "rehomingIntervalMin" : 60,
  "rehomingIntervalMax" : 120,
  "voicePortalNumber" : null,
  "voicePortalExtension" : null
}

Reply:
Removed the dummy values. Will refactor other fields coming from Mobius api response in a separate PR as discussed with Priya.